addons/account/tests/test_account_payment_register.py

```diff
@@ -649,15 +649,9 @@ def test_register_payment_multi_batches_grouped(self):
             .with_context(active_model='account.move', active_ids=active_ids)\
             .create({
                 'group_payment': True,
-                'amount': 1000.0
```

Comment:
Removed it as we were already not supposed to be able to change the amount on group_payments.
And now we take the amount into account, as it can change with installments